src/AdminLTETemplatesServiceProvider.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace InfyOm\AdminLTETemplates;
+namespace InfyOm\Generator;
```

Comment:
why this change?

Reply:
It is only a namespace change, but I do want to know the reason behind the change @tiago-wd

Reply:
@cyril-sf Is tiago's response answering your question or would you like a more described answer around the change?